File: app/src/main/java/com/example/medicinesearch/MainWindow.java
package com.example.medicinesearch;

import android.os.Bundle;
import android.view.MenuItem;

import androidx.annotation.NonNull;
import androidx.appcompat.app.AppCompatActivity;
import androidx.fragment.app.Fragment;

import com.google.android.material.bottomnavigation.BottomNavigationView;

public class MainWindow extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main_window);

        BottomNavigationView bottomNav = findViewById(R.id.bottom_navigation);
        bottomNav.setOnNavigationItemSelectedListener(navListener);

        if (savedInstanceState == null) {
            getSupportFragmentManager()
                    .beginTransaction()
                    .add(R.id.fragment_container, new Home())
                    .commit();
        }
    }

    private BottomNavigationView.OnNavigationItemSelectedListener navListener =
            new BottomNavigationView.OnNavigationItemSelectedListener() {
                @Override
                public boolean onNavigationItemSelected(@NonNull MenuItem item) {
                    Fragment selectedFragmet = null;

                    switch (item.getItemId()) {
                        case R.id.nav_home:
                            selectedFragmet = new Home();
                            break;

                        case R.id.nav_royhat:
                            selectedFragmet = new Избранное();
                            break;

                        case R.id.nav_karzina:
                            selectedFragmet = new Корзина();
                            break;

                        case R.id.nav_location:
                            selectedFragmet = new Карта();
                            break;

                        case R.id.nav_menu:
                            selectedFragmet = new Профил();
                            break;
                    }
                    getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container, selectedFragmet).commit();
                    return true;
                }
            };
}

File: app/src/main/java/uz/company/medicinesearch/MainWindow.kt
package uz.company.medicinesearch

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import androidx.fragment.app.Fragment

import com.google.android.material.bottomnavigation.BottomNavigationView

class MainWindow : AppCompatActivity() {


    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main_window)

        val bottomNav = findViewById<BottomNavigationView>(R.id.bottom_navigation)
        bottomNav.setOnNavigationItemSelectedListener(navListener)
        if (savedInstanceState == null) {
            supportFragmentManager
                    .beginTransaction()
                    .add(R.id.fragment_container, Home())
                    .commit()
        }


    }

    private val navListener = BottomNavigationView.OnNavigationItemSelectedListener { item ->
        var selectedFragmet: Fragment? = null
        when (item.itemId) {
            R.id.nav_home -> selectedFragmet = Home()
            R.id.nav_royhat -> selectedFragmet = Избранное()
            R.id.nav_karzina -> selectedFragmet = Корзина()
            R.id.nav_location -> selectedFragmet = Карта()
            R.id.nav_menu -> selectedFragmet = Профил()
        }
        supportFragmentManager.beginTransaction().replace(R.id.fragment_container, selectedFragmet!!).commit()
        true
    }


}